entity-service/src/main/kotlin/com/egm/stellio/entity/model/UpdateResult.kt

```diff
@@ -1,11 +1,39 @@
 package com.egm.stellio.entity.model
 
-data class UpdateResult(
```

Comment:
I'm worried by the fact that it breaks the datatype names defined in the specification.